nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/provenance/ui/provenance-event-listing/provenance-event-table/provenance-event-table.component.html

```diff
@@ -15,7 +15,8 @@
   ~ limitations under the License.
   -->
 
-<div class="provenance-event-table h-full">
+<div class="provenance-event-table h-full flex flex-col gap-y-2">
+    <error-banner></error-banner>
```

Comment:
As this banner is used for both provenance and lineage issues, we might need to clear them when we switch from one view to the other.